### DISCUSSION TITLE
Mapping RSD metadata to schema.org

### DISCUSSION THREAD

**Discussion by nuest** [research-software-directory/RSD-as-a-service#957] [Q&A]
Has this been discussed already, or did someone work on this?

We're currently discussing whether to use https://schema.org/SoftwareApplication or https://schema.org/SoftwareSourceCode as the base for software (I'm leaning towards the latter), and maybe there already exists some mapping that can help us make that call.

_Thanks!_

**Comment by jmaassen** [comment]
Indeed the latter makes slightly more sense to me. At least for open source application where the RSD has the repo URL, programming languages, etc. The SoftwareApplication seems better suited to describe binary distributions, so this may be a better option for closed source applications (although we do not have many of those in the RSD). 

If I read the schema correctly, a SoftwareSourceCode can also contain a SoftwareApplication that further describes the final product, so combining them could also be an option?

**Reply by nuest** [reply]
Yes, combining the is of course also an option.

For the fields that we need the mapping is pretty straightforward. I'll take a mental not to share our result again here, but am still open to someone posting a ready-to-use Python library that has this feature ;-)
